fvid/fvid.py

```diff
@@ -218,6 +219,10 @@ def main():
         save_bits_to_file("./", bits)
 
     elif args.encode:
+        # isdigit has the benefit of being True and raising an error if the user passes a negative string
+        # all() lets us check if both the negative sign and forward slash are in the string, to prevent negative fractions
+        if (not args.framerate.isdigit() and "/" not in args.framerate) or all(x in args.framerate for x in ("-", "/")):
```

Comment:
Really cool :)

Reply:
Thanks! I could probably condense it into a single check, but that'd be harder and less readable ;)